### PR TITLE
Improve disabled button rendering #70

### PR DESCRIPTION
**Pull request by basix86**

### CODE REVIEW

File: flatlaf-core/src/main/java/com/formdev/flatlaf/FlatLaf.java [hunk 1]
```diff
@@ -36,20 +36,13 @@
...
-import javax.swing.JLabel;
-import javax.swing.JRootPane;
-import javax.swing.JTabbedPane;
-import javax.swing.LookAndFeel;
-import javax.swing.PopupFactory;
-import javax.swing.SwingUtilities;
-import javax.swing.UIDefaults;
-import javax.swing.UIManager;
-import javax.swing.UnsupportedLookAndFeelException;
+import javax.swing.*;
```
Comment (smileatom) [inline flatlaf-core/src/main/java/com/formdev/flatlaf/FlatLaf.java]: This is probably not desired.  Explicit imports is preferred.

File: flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatUIUtils.java [hunk 2]
```diff
@@ -16,28 +16,19 @@
...
-import java.awt.Container;
-import java.awt.Dimension;
-import java.awt.Font;
-import java.awt.Graphics;
-import java.awt.Graphics2D;
-import java.awt.Insets;
-import java.awt.Rectangle;
-import java.awt.RenderingHints;
-import java.awt.Shape;
+import java.awt.*;
```
Comment (smileatom) [inline flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatUIUtils.java]: Again likely undesirable

Comment (smileatom) [inline flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatUIUtils.java]: Again likely undesirable

File: flatlaf-core/src/main/java/com/formdev/flatlaf/FlatLaf.java (hunk 1 above)
Comment (basix86) [reply flatlaf-core/src/main/java/com/formdev/flatlaf/FlatLaf.java]: ok, I wait your codestyle
https://github.com/JFormDesigner/FlatLaf/issues/71

**Comment by DevCharly** [comment]
Thanks for the PR. I'll test it next week because I'm out-of-office until Wednesday.

File: flatlaf-core/src/main/java/com/formdev/flatlaf/FlatLaf.java (hunk 1 above)
Comment (smileatom) [reply flatlaf-core/src/main/java/com/formdev/flatlaf/FlatLaf.java]: Although .* imports seems like a code style issue.. Its not especially in the case of root level UI code like L&F.  This should be reverted.

**Pull request merged**